src/main/java/com/learntechfree/ltf/entity/ArticleRevision.java

```diff
@@ -0,0 +1,47 @@
+package com.learntechfree.ltf.entity;
+
+
+import lombok.*;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.*;
+
+
+@Entity
+@Table(name = "article_revisions")
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class ArticleRevision extends AuditModel {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column
+    private String title;
+
+    @Column
+    private String subTitle;
+
+    @Column(nullable = false)
+    private String slug;
```

Comment:
@shrijan00003  The slug will be determined with the initial title of the post and let's not change it or save it every time because sometimes title needs to be refactored after a post is submitted but the URL should remain same to avoid an issue like 404.